python/ray/_private/worker.py

```diff
@@ -1126,6 +1126,7 @@ def init(
     namespace: Optional[str] = None,
     runtime_env: Optional[Union[Dict[str, Any], "RuntimeEnv"]] = None,  # noqa: F821
     storage: Optional[str] = None,
+    no_gcs: bool = False,
```

Comment:
We shouldn't change the API.

Reply:
should we have a global variable then? initialization order might be tricky

Reply:
So the problem is that some tests don't start GCS, and then the newly added RPC hangs. Also, monitor uses this for now as it seems GCS is not always available to it.